hack/common.sh

```diff
@@ -55,6 +55,8 @@ readonly OS_CROSS_COMPILE_PLATFORMS=(
 readonly OS_CROSS_COMPILE_TARGETS=(
   cmd/openshift
   cmd/oc
+  pkg/sdn/plugin/sdn-cni-plugin
```

Comment:
@smarterclayton only OS_CROSS_COMPILE_TARGETS gets used for OS_ALL_TARGETS which gets used to build the actual binaries in os::build::export_targets().  Without putting these new binaries in CROSS_COMPILE_TARGETS they don't get build with 'make' or hack/build-go.sh.

Where do they need to get put to be built at the same time as cmd/openshift?